storm-core/src/clj/org/apache/storm/cluster.clj

```diff
@@ -73,6 +73,7 @@
   (teardown-topology-errors! [this storm-id])
   (heartbeat-storms [this])
   (error-topologies [this])
+  (backpressure-topologies [this])
```

Comment:
I agree. I think, however, it is *somewhat* consistent with the interface (e.g. heartbeat-storms, error-topologies don't say get-X-topologies-id). Additionally, I think we should call them one thing throughout (is it storms? is it topologies? is it keys?) Happy to make that change, but I think it's bigger than this PR. Thoughts?